docs/index.rst

```diff
@@ -0,0 +1,83 @@
+OMERO.web installation and maintenance
+======================================
+
+OMERO.web is a Python 3 client of the OMERO platform that provides a
+web-based UI and JSON API.
+This section provides links to detailed step-by-step walkthroughs describing how to install, customize, maintain and run OMERO.web for several systems.
+OMERO.web is installed **separately** from the OMERO.server.
+
+OMERO.web can be deployed with:
+
+-  `WSGI <https://wsgi.readthedocs.org>`_ using a WSGI capable web server
+   such as
+   `NGINX <https://nginx.org/>`_ and `Gunicorn <https://docs.gunicorn.org/>`_
+-  the built-in Django lightweight development server. This type of deployment should only be used for **testing** purpose
+   only; see the :omerodoc:`Developers Deployment <developers/Web/Deployment.html>` page.
+
+If you need help configuring your firewall rules, see
+:omerodoc:`Security <sysadmins/server-security.html>` for more details.
+
+Depending upon which platform you are using, you may find a
+more specific walkthrough listed below. The guides use the example of deploying OMERO.web with
+`NGINX <https://nginx.org/>`_ and `Gunicorn <https://docs.gunicorn.org/>`_.
+OMERO can automatically generate a
+configuration file for your webserver. The location of the file will depend
+on your system, please refer to your webserver's manual. See in the section `Customizing your OMERO.web installation` in the various walkthroughs for more options.
+
+.. _omero_web_configuration:
+
+Configuration
+-------------
+You will find in the various guides how to create the NGINX OMERO configuration file
+and the configuration steps for the NGINX and Gunicorn.
+Advanced Gunicorn setups are also described to enable the download of binary data
+and to handle multiple clients on a single worker
+thread switching context as necessary while streaming binary data from
+OMERO.server. Depending on the traffic and scale of the repository you should
+configure connections and speed limits on your server to avoid blocking
+resources.
+
+Walkthroughs
+------------
+
+**Recommended:**
+
+
+:doc:`walkthrough/omeroweb-install-centos7-ice3.6`
+  Instructions for installing OMERO.web from scratch on CentOS 7 with Ice 3.6.
+
+:doc:`walkthrough/omeroweb-install-debian10-ice3.6`
+  Instructions for installing OMERO.web from scratch on Debian 10 with Ice 3.6.
```

Comment:
A bit strange that we list Debian 10 under "Recommended" and Debian 9 under "Upcoming". Shouldn't it be the other way round?

Reply:
This is a ``copy & paste`` from ome-doc, we can certainly fix it.
The approach proposed here will simplify the release of such change at no cost.